charter.md

```diff
@@ -72,28 +78,151 @@ this works.
 
 ### Vacancies
 
-In the event of a resignation or other loss of an elected steering committee
-member, the candidate with the next most votes from the previous election will
-be offered the seat.  This process will continue until the seat is filled.
+In the event of a resignation or other loss of an elected committee member, the
+next most preferred candidate from the previous election will be offered the
+seat.
+
+A maximum of one (1) committee member may be selected this way between
+elections.
 
 In case this fails to fill the seat, a special election for that position will
-be held as soon as possible. [Eligible voters][voter-eligibility] from the most
-recent election will vote in the special election i.e., eligibility will not be
-redetermined at the time of the special election. A committee member elected in
-a special election will serve out the remainder of the term for the person they
-are replacing, regardless of the length of that remainder.
+be held as soon as possible.
+
+[Eligible voters][voter-eligibility] from the most recent election will vote in
+the special election i.e., eligibility will not be redetermined at the time of
+the special election.
+
+A committee member elected in a special election will serve out the remainder
+of the term for the person they are replacing, regardless of the length of that
+remainder.
 
+[maximal-representation]: /elections.md#maximal-representation
 [voter-eligibility]: /elections.md#eligibility-for-voting
 
-## Quorum
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Removal
+
+#### No confidence
+
+A Steering Committee member may be removed by an affirmative vote of a
+**_three-quarters supermajority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
```

Comment:
I'd like to suggest we remove all examples from this document. They are helpful and illustrative when debating, but now that we have specific language, I'd suggest we remove the examples.

Reply:
I think the examples are vital for making it 100% clear the intent of the words.

Reply:
I agree they should be kept to keep the meaning unambiguous

Reply:
Is there language we could add or modify as far as the in-force language that would make the example unnecessary?

I feel like if we need an illustrative example in addition to the in-force language, that we might not have done a good enough job with the in-force language.. we shouldn't need to state the same thing twice for it to be clear.

Reply:
Stating the same thing two (2) times in different ways is very common in documents like these in my experience. It demonstrates that the author didn't make a mistake.

Reply:
Do you have examples of where this is done in a charter document like this?

Reply:
Not off the top of my head, sorry. I don't understand why you think it is bad to include them? This isn't go code, redundancy doesn't seem bad, especially since there's no compiler :)

Reply:
My concern would be, if you state the similar things twice but in different words, it leaves room for dispute on which words are actually the in-force language.

If there was a way to convey the information fully as one cohesive statement that didn't require saying it a second time, there's less opportunity for dispute in the future.

Reply:
Yes, it would be very bad if the example didn't match the text. But the examples do match the text, so we won't have that problem?

Reply:
If we have clear in-force text, then an example shouldn't be required.

Reply:
I'm going to opt to resolve this convo on examples.
It sounds like the examples help remove ambiguity for readers (which was my hope).

If we want to come back at a future date and further strengthen the in-force text to remove examples, it should roughly be a no-op (intent-wise).